docs/Easystack-files.rst

```diff
@@ -32,68 +39,80 @@ To build software with *Easystack*, type:
 
 where ``example.yaml`` is the file with specifications that you just created (more about this in the next section).
 
+
 .. _easystack_structure:
 
 Structure of an easystack file
 ------------------------------
 
 Easystack files are written in `YAML syntax <https://learnxinyminutes.com/docs/yaml>`_.
 
-Essentially, easystack files list the EasyConfigs you want to install. These are listed under the 'easyconfigs:' key:
+Essentially, an easystack file lists the easyconfig files you want to install, which are specified under the
+`easyconfigs` key.
+
+For example:
 
 .. code:: yaml
 
   easyconfigs:
     - PyTorch-1.12.0-foss-2022a-CUDA-11.7.0.eb
     - OpenFOAM-v2206-foss-2022a.eb
 
-(note the use of the ``-`` to list the items).
+.. note::
+   You must use '``-``' to list the easyconfigs in an easystack file.
+
+In addition, you can specify additional configuration options via the ``options`` subkey
+which will only apply to the installation of a particular easyconfig file.
 
-In addition, you can specify additional options that will only apply to the installation of that EasyConfig:
+For example:
 
 .. code:: yaml
 
   easyconfigs:
     - PyTorch-1.12.0-foss-2022a-CUDA-11.7.0.eb:
-      options: {
-        'from-pr': 15924,
-        'debug': True
-      }
```

Comment:
We use the `{}` braces in `test/framework/easystack/test_easystack_basic.yaml`. I _guess_ you tested, and it works without braces as well, but then maybe we should remove it from all of the `easystack` test files too? I think it would be good to test _exactly_ what we _claim_ one should do in the docs

Reply:
Oh, and I see you also removed the quotes around the `'from-pr'`. I guess the same reasoning applies there: I'm ok with it (and this is actually simpler), as long as that's also what we use in the tests.

Reply:
As mentioned in the description, I've pushed https://github.com/easybuilders/easybuild-framework/commit/b93dd22ffd710fcbe5f306aa52b4e03c64f5a039 into the PR so we're testing both syntaxes (with and without `{`/`}` and quotes)

Reply:
Lazy me jumped immediately to the changes without reading the comment, sorry. Indeed, perfect like this then.